<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Kết nối thiết bị</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QComboBox" name="comportnameBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Kết nối</string>
     </property>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="danhmuctreeWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>141</width>
      <height>251</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Danh mục</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Dầu gội đầu</string>
     </property>
     <item>
      <property name="text">
       <string>Clear bạc hà</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Xmen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Head &amp; Shoulder</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Romano</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Kem đánh răng</string>
     </property>
     <item>
      <property name="text">
       <string>PS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Congate</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Quần áo</string>
     </property>
     <item>
      <property name="text">
       <string>Việt tiến</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Việt Pháp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lacos</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Giày dép</string>
     </property>
     <item>
      <property name="text">
       <string>Nike</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thượng đình</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Adidas</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Trái cây</string>
     </property>
     <item>
      <property name="text">
       <string>Sầu riêng</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Măng cục</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Xòai</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dưa hấu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bưởi</string>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>211</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Cài đặt thông tin</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QFrame" name="displayframe">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>172</width>
       <height>72</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>172</width>
       <height>72</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>172</width>
       <height>72</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(59, 59, 59);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="vendorlineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Tên Hãng sản xuất</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="brandlineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Tên nhãn hiệu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newpricelineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Giá bán hiện tại</string>
     </property>
    </widget>
    <widget class="QComboBox" name="iconBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Biểu tượng</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cập nhật</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="oldpricelineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Giá bán trước đây</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
